src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./Introduce.scss";
+import { useNavigate } from "react-router-dom";
+
+const IntroduceName = () => {
+  const navigate = useNavigate();
+
+  const logoname = "> wecode";
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+  const handleButton = (e) => {
+    e.preventDefault();
+    navigate("/product");
+    console.log("button 태그 동작");
+  };
+
+  return (
+    <div className="introducename">
```

Comment:
```suggestion
    <div className="introduce">
```
위의 리뷰와 연관지으면 이렇게 구현할 수 있겠네요!

Reply:
넵!